Nt/custom filters

noumantahir opened the pull request:
Added support for customizing home screen tabs. tabs default to friends, communities and hot. using redux to hot swap tabs

### Known issues
Having issues loading initial posts saved in store on startup as it was set to fixed tab before, will come up with a clean solution instead of messing up structure, for now it does not affect normal app operation

### Screenshots/Video
https://user-images.githubusercontent.com/6298342/115639329-bd512680-a32d-11eb-8fac-49e5f74167e3.mov

feruzm commented on src/components/customiseFiltersModal/customiseFiltersModal.tsx:
```diff
@@ -0,0 +1,142 @@
+    //save snippet based on editor type
+    const _onApply = () => {
+        if(selectedFilters.size !== 3){
+            alert("Please select exactly three filters");
```
please add to translation

noumantahir commented:
Updates:

- Modal tweaks
- using new promoted posts response with post_data
- fixed account swap data reload issue
- fixed empty view issues caused by redux
